### DISCUSSION TITLE
Selecting custom voice banks.

### DISCUSSION THREAD

**Discussion by Ernisius** [probonopd/MiniDexed#666] [Q&A] [answered]
I just built this and it all works except I can only select 32 voices. I've copied the sysexe data to /Sysexe/voice, but I can't find how to select them.
Selecting Bank does nothing. Am I missing something?

**Comment by probonopd** [comment, marked as answer]
`/Sysexe/voice` is not correct. The files need to go to `sysex/voice/` and must be named `000001_rom3a.syx`,... as https://github.com/probonopd/MiniDexed/blob/main/getsysex.sh does for you.

**Comment by Ernisius** [comment]
Thanks for the fast response. Can't believe I made such a dumb mistake.
